readme.ipynb

```diff
@@ -49,8 +49,9 @@
     "def func():\n",
```

Comment:
<p>Where is my previous comment?!?</p><p>Line 2: change the name of the variable</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/epalese/reviewnb-test/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>